client/src/shared/components/Tabs/Tabs.styled.ts

```diff
@@ -7,33 +7,46 @@ export const StyledTabs = styled.div`
 
 export const StyledTabsList = styled.div`
   display: flex;
-  border-radius: 12.75px;
-  background-color: #ececf0;
-  padding: 4px;
+  background-color: transparent;
+  position: relative;
+
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    height: 3px;
+    background-color: ${({ theme }) => theme.colors.gray900};
+    transition: all 0.3s cubic-bezier(0.4, 0, 0.2, 1);
+    transform: translateX(calc(var(--active-tab-index, 0) * 100%));
+    width: calc(100% / var(--tab-count, 1));
+  }
 `;
 
 export const StyledTabsTrigger = styled.button`
+  border-radius: 0;
+  padding: 16px 20px;
+  font-size: 16px;
+  font-weight: 600;
   width: 100%;
-  border-radius: 10px;
-  padding: 6px 12px;
-  font-size: 14px;
-  font-weight: 500;
   background: transparent;
   border: none;
   cursor: pointer;
-  transition: all 0.2s ease;
-  color: #64748b;
+  transition:
+    color 0.3s cubic-bezier(0.4, 0, 0.2, 1),
+    background-color 0.3s cubic-bezier(0.4, 0, 0.2, 1);
+  color: ${({ theme }) => theme.colors.gray500};
 
   &:hover {
-    color: #334155;
+    color: ${({ theme }) => theme.colors.gray700};
+    background-color: ${({ theme }) => theme.colors.primary50};
```

Comment:
p2.
탭의 기본 색상이 검정인데, 현재 호버 색상이 푸른 계열이라 다소 튀는 느낌이 있는 것 같아요. 톤의 통일감을 위해 `gray50`이나 `gray100` 정도의 색상으로 조정해보는 건 어떠신가요??
```suggestion
    background-color: ${({ theme }) => theme.colors.gray100};
```

Reply:
동의합니다. 디자인 적으로 크게 감각이 없어 인지하지 못했네용

3fe350189a4f8a502e33b9db56bb53bbb0eb7239